exercises/camicia/canonical-data.json

```
@@ -0,0 +1,215 @@
+      "property": "simulateGame",
+      "input": {
+        "playerA": "A-QK------Q----KA-----J---",
+        "playerB": "-JAK----A--Q----J---QJ--K-"
+      },
+      "expected": {
+        "status": "finished",
+        "cards": 6913, 
+        "tricks": 960
+      }
+    },
+    {
+      "uuid": "c90b6f8d-7013-49f3-b5cb-14ea006cca1d",
+      "description": "Mann and Wu 2007",
+      "property": "simulateGame",
+      "input": {
+        "playerA": "K-KK----K-A-----JAA--Q--J-",
+        "playerB": "---Q---Q-J-----J------AQ--"
+      },
+      "expected": {
+        "status": "finished",
+        "cards": 7157, 
+        "tricks": 1007
+      }
+    },
+    {
+      "uuid": "a3f1fbc5-1d0b-499a-92a5-22932dfc6bc8",
+      "description": "Nessler 2012",
+      "property": "simulateGame",
+      "input": {
+        "playerA": "----Q------A--K--A-A--QJK-",
+        "playerB": "-Q--J--J---QK---K----JA---"
+      },
+      "expected": {
+        "status": "finished",
+        "cards": 7207, 
+        "tricks": 1015
+      }
+    },
+    {
+      "uuid": "9cefb1ba-e6d1-4ab7-9d8f-76d8e0976d5f",
+      "description": "Anderson 2013",
+      "property": "simulateGame",
+      "input": {
+        "playerA": "--A-Q--J--J---Q--AJ-K---K-",
+        "playerB": "-J-------Q------A--A--QKK-"
+      },
+      "expected": {
+        "status": "finished",
+        "cards": 7225, 
+        "tricks": 1016
+      }
+    },
+    {
+      "uuid": "d37c0318-5be6-48d0-ab72-a7aaaff86179",
+      "description": "Rucklidge 2014",
+      "property": "simulateGame",
+      "input": {
+        "playerA": "-J------Q------AAA-----QQ-",
+        "playerB": "K----JA-----------KQ-K-JJK"
+      },
+      "expected": {
+        "status": "finished",
+        "cards": 7959,
+        "tricks": 1122
+      }
+    },
+    {
+      "uuid": "4305e479-ba87-432f-8a29-cd2bd75d2f05",
+      "description": "Nessler 2021",
+      "property": "simulateGame",
+      "input": {
+        "playerA": "----K---A--Q-A--JJA------J",
+        "playerB": "-----KK---------A-JK-Q-Q-Q"
+      },
+      "expected": {
+        "status": "finished",
+        "cards": 7972, 
+        "tricks": 1106
+      }
+    },
+    {
+      "uuid": "252f5cc3-b86d-4251-87ce-f920b7a6a559",
+      "description": "Nessler 2022",
+      "property": "simulateGame",
+      "input": {
+        "playerA": "---AJ--Q---------QAKQJJ-QK",
+        "playerB": "-----A----KJ-K--------A---"
+      },
+      "expected": {
+        "status": "finished",
+        "cards": 8344, 
+        "tricks": 1164
+      }
+    },
+    {
+      "uuid": "b9efcfa4-842f-4542-8112-8389c714d958",
+      "description": "Casella 2024, first infinite game found",
+      "property": "simulateGame",
+      "input": {
+        "playerA": "---K---Q-KQAJ-----AAJ--J--",
+        "playerB": "----------Q----KQ-J-----KA"
+      },
+      "expected": {
+        "status": "loop",
+        "cards": 474, 
+        "tricks": 66
+      }
+    }
+  ]
+}
```

```suggestion
}

```

All lines in text should be terminated by an EOL (end of line) character.

---

Is it normal for CI to fail anyway?

---

No. Those need to be addressed at some point, but those other to be relatively easy to track down and deal with.

---

There should be no more EOF errors.
I don't think there are any other CI issues that I can easily resolve.

---

[The JSON formatter](https://github.com/exercism/problem-specifications/actions/runs/17361274258/job/49281705328?pr=2581) asks you to run pretty or to apply the given diff. You can copy/paste the diff and apply it, or copy the `+` output and use that.
The link checker has issue with an unrelated exercise. That should be dealt with outside of this PR.

---

Ok, I'll try

---

How did you do that? So I can fix it too before pushing the new tests.